modules/icinga/templates/host.erb

```diff
@@ -4,5 +4,7 @@ define host {
         address      <%= @address %>
         display_name <%= @display_name %>
         host_name    <%= @hostalias %>
-        hostgroups   all
+        hostgroups   all<% if @parents %>
+        parents      <%= @parents %>
+        <% end %>
```

Comment:
This could be more clearly written as:

```
hostgroups   all
<%- if @parents -%>
parents      <%= @parents %>
<%- end -%>
```

The hyphens on the edge of the ERB blocks ensure that the lines containing the ERB don't show up in the file.
```